rollup.config.js

```diff
@@ -227,7 +227,8 @@ const target_debug = {
         file: 'build/playcanvas.dbg.js',
         format: 'umd',
         indent: '\t',
-        name: 'pc'
+        name: 'pc',
+        sourcemap: true
```

Comment:
Again, this isn't really related to this PR. But @yaustar, you've done a lot of playing with source maps. Does this look good to you? What does this do? Generate a source map alongside `playcanvas.dbg.js`? Or embed the source maps inside `playcanvas.dbg.js`?

Reply:
@kungfooman Can you help with this question please?

Reply:
I addressed the source maps issues in https://github.com/playcanvas/engine/pull/4293, so hopefully we're all good. I'll remove this line from the PR.